dev/validate_bom__config.py

```diff
@@ -1501,9 +1501,12 @@ def add_config(self, options, script):
                     ' --push-gateway {gateway}'
                     .format(gateway=options.monitoring_prometheus_gateway))
     if options.monitoring_filter_dir:
-      script.append('sudo mkdir -p /opt/spinnaker-monitoring/filters')
-      script.append('cat monitoring_filter.tz'
-                    ' | (cd /opt/spinnaker-monitoring/filters; sudo tar xf -)')
+      # Unpack the tar file into halyard's config directory.
+      script.extend([
+          'mkdir -p ~/.hal/default/profiles/monitoring-daemon/filters',
```

Comment:
There was a `sudo` dropped here, still needed?

Reply:
Before it was dropping directly into the install directory under /opt.
Now it is dropping into halyards config directory, under $HOME, where halyard can pick it up.
Recent versions of halyard know about this.